.circleci/config.yml

```diff
@@ -162,7 +162,13 @@ jobs:
             cf api https://api.fr.cloud.gov
             cf auth "$CF_USERNAME" "$CF_PASSWORD"
             cf target -o $CF_ORG -s "$CF_SPACE"
-            ./deploy-cloudgov-docker.sh rolling test $CGHOSTNAME_BACKEND $CGHOSTNAME_FRONTEND $DOCKER_IMAGE_BACKEND $DOCKER_IMAGE_FRONTEND $CIRCLE_BRANCH
+      - run:
+          name: "Deploy clamav-rest"
```

Comment:
Decoupled these deployments with the intention that we eventually will be able to update each step to only re-deploy if there was a change to the specific service.